src/components/views/rooms/wysiwyg_composer/hooks/usePlainTextListeners.ts

```diff
@@ -97,10 +98,11 @@ export function usePlainTextListeners(
                 setContent(text);
                 onChange?.(text);
             } else if (isNotNull(ref) && isNotNull(ref.current)) {
-                // if called with no argument, read the current innerHTML from the ref
+                // if called with no argument, read the current innerHTML from the ref and amend it as per `onInput`
                 const currentRefContent = ref.current.innerHTML;
-                setContent(currentRefContent);
-                onChange?.(currentRefContent);
+                const amendedContent = amendInnerHtml(currentRefContent);
+                setContent(amendedContent);
+                onChange?.(amendedContent);
```

Comment:
This was an error I picked up working in the area, the content of the composer needs to be run through `amendInnerHtml` before being stored in the `content` state.